### PR TITLE
client: remove Experimental prefix from AddSSTable command

### PR DESCRIPTION
**Pull request by danhhz**
We haven't found any consistency issues in testing and all the
correctness TODOs have been done.

### CODE REVIEW

**Review by tbg — APPROVED** [review]

File: build/teamcity-nightly-acceptance.sh [hunk 1]
```diff
@@ -88,7 +88,7 @@ pkg/acceptance/prepare.sh
 # TODO(cuongdo): give this a unique name, to allow multiple local runs of the
 # nightlies to execute simultaneously
 mkdir -p artifacts/acceptance
-export TMPDIR=$PWD/artifacts/acceptance
+export TMPDIR=/tmp
```
Comment (benesch) [inline build/teamcity-nightly-acceptance.sh]: This looks to have slipped in.

Comment (danhhz) [reply build/teamcity-nightly-acceptance.sh]: argh! this is not going to be the last time that slips in

**Comment by benesch** [comment]
It occurs to me that you should remove "experimental" from the cluster setting too.

**Comment by tschottdorf** [comment]
Renaming a cluster setting isn't necessarily trivial. I'd do this separately.

**Comment by benesch** [comment]
Fine with me, but if we want to rename it for 1.1 the sooner the better.

**Comment by danhhz** [comment]
@benesch agreed. Wanna PR it? It's a hidden setting and it's only been present in alphas, so I don't think there are any backward compatibility concerns.

**Comment by danhhz** [comment]
TFTRs!

**Pull request merged**